webclient/package.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
I can't found anything about that, but we could create a script in the first package.json that could run "npm install; cd webclient; npm install; cd ..".
But if we do that, it means that instead of running "npm install" as usual to install the dependencies, we will run "npm run install", and I'm not sure if it's a good practice.